tests/phpunit/elementor/modules/global-classes/test-global-classes-usage.php

```diff
@@ -0,0 +1,260 @@
+					'settings' => [
+						'classes' => [
+							'value' => ['g-4-124', 'e-4-1222']
+						]
+					],
+					'elements' => [
+						[
+							'id' => 'ghi7894',
+							'elType' => 'e-button',
+							'settings' => [
+								'classes' => [
+									'value' => ['g-4-123', 'e-4-1222']
+								]
+							],
+						],
+						[
+							'id' => 'jkl1014',
+							'elType' => 'widget',
+							'widgetType' => 'heading',
+							'settings' => [],
+						]
+					],
+				],
+			],
+		],
+	];
+
+	public function setUp(): void {
+		parent::setUp();
+
+		Plugin::$instance->kits_manager->create_new_kit( 'kit' );
+
+		Global_Classes_Repository::make()->put(
+			$this->mock_global_classes['items'],
+			$this->mock_global_classes['order'],
+		);
+
+		remove_all_filters( 'elementor/tracker/send_tracking_data_params' );
+	}
+
+	public function tear_down() {
+		( new Global_Classes_Repository() )->put( [], [] );
+
+
+		foreach ( $this->post_ids as $post_id ) {
+			wp_delete_post( $post_id, true );
+		}
+
+		$this->post_ids = [];
+
+		parent::tear_down();
+	}
+
+	public function test_global_classes_usage() {
+		// Arrange.
+		$global_classes_usage = new Global_Classes_Usage();
+		$global_classes_usage->register_hooks();
+		$this->make_mock_post_with_elements( $this->mock_elementor_data );
+
+		// Act.
+		$params = [];
+
+		$params = apply_filters( 'elementor/tracker/send_tracking_data_params', $params );
+
+		// Assert.
+		$this->assertEquals( 2, $params['usages']['global_classes']['total_count'] );
+		$this->assertEquals(
+			[
+				'e-div-block' => 3,
+				'e-heading' => 1,
+			],
+			$params['usages']['global_classes']['applied_classes_per_element_type'],
+		);
+	}
+
+	public function test_global_classes_usage_with_multiple_posts() {
+		// Arrange.
+		$global_classes_usage = new Global_Classes_Usage();
+		$global_classes_usage->register_hooks();
+
+		$this->make_mock_post_with_elements( $this->mock_elementor_data );
+		$this->make_mock_post_with_elements( $this->mock_elementor_data_2 );
+
+		// Act.
+		$params = [];
+
+		$params = apply_filters( 'elementor/tracker/send_tracking_data_params', $params );
+
+		// Assert.
+		$this->assertEquals( 2, $params['usages']['global_classes']['total_count'] );
+		$this->assertEquals(
+			[
+				'e-div-block' => 5,
+				'e-heading' => 1,
+				'e-button' => 1,
+			],
+			$params['usages']['global_classes']['applied_classes_per_element_type'],
+		);
+	}
+
+	public function test_global_classes_usage_with_no_classes_exist() {
+		// Arrange.
+		Global_Classes_Repository::make()->put( [], [] );
+		$global_classes_usage = new Global_Classes_Usage();
+		$global_classes_usage->register_hooks();
+
+		// Act.
+		$params = [];
+
+		$params = apply_filters( 'elementor/tracker/send_tracking_data_params', $params );
+
+		// Assert.
+		$this->assertEquals( 0, $params['usages']['global_classes']['total_count'] );
+		$this->assertArrayNotHasKey( 'applied_classes_per_element_type', $params['usages']['global_classes'] );
```

Comment:
```suggestion
		$this->assertSame( [ 'total_count' => 0 ], $params['usages']['global_classes'] );
```

This way you don't need to check twice + don't need to change the test when something new will be added